framework/base/BaseObject.php

```diff
@@ -124,12 +124,11 @@ public function __get($name)
         $getter = 'get' . $name;
         if (method_exists($this, $getter)) {
             return $this->$getter();
-        }
-        if (method_exists($this, 'set' . $name)) {
+        } elseif (method_exists($this, 'set' . $name)) {
```

Comment:
No need to change it to `elseif` since there was return.

Reply:
Shouldn't `__get()` and `__set()` use the same coding style for `if` / `elseif`? This wasn't the case when I started so I thought, let's unify the style:

```
    public function __get($name)
    {
        $getter = 'get' . $name;
        if (method_exists($this, $getter)) {
            return $this->$getter();
        }
        if (method_exists($this, 'set' . $name)) {
            throw new InvalidCallException('Getting write-only property: ' . get_class($this) . '::' . $name);
        }
        throw new UnknownPropertyException('Getting unknown property: ' . get_class($this) . '::' . $name);
    }

```

```
    public function __set($name, $value)
    {
        $setter = 'set' . $name;
        if (method_exists($this, $setter)) {
            $this->$setter($value);
        } elseif (method_exists($this, 'get' . $name)) {
            throw new InvalidCallException('Setting read-only property: ' . get_class($this) . '::' . $name);
        } else {
            throw new UnknownPropertyException('Setting unknown property: ' . get_class($this) . '::' . $name);
        }
    }
```

Reply:
Also please note PSR-2: https://www.php-fig.org/psr/psr-2/#51-if-elseif-else

> The keyword elseif SHOULD be used instead of else if so that all control
keywords look like single words.

Reply:
I didn't mean `else if`, I mean `if`. `else` isn't needed at all in this case. Yes, it's good to unify it removing useless constructs.

Reply:
Ok, I understand that. I'm pretty sure that some PSR recommended to *always* use a `else` / `elseif` block even if the block before ended with a `return`, as the latter is easy to miss and the code is more readable.

But whatever - I'll undo everything.

Reply:
Nope. No PSR recommends that.